src/mysql_shell/__init__.py

```diff
@@ -64,40 +64,36 @@ def _run_code(self, code: str) -> None:
         template = _jinja_env.get_template("try_except_wrapper.py.jinja")
         error_file = self._container.path("/tmp/mysqlsh_error.json")
 
-        def render(connection_info: "relations.database_requires.ConnectionInformation"):
-            return template.render(
-                username=connection_info.username,
-                password=connection_info.password,
-                host=connection_info.host,
-                port=connection_info.port,
-                code=code,
-                error_filepath=error_file.relative_to_container,
-            )
-
-        # Redact password from log
-        logged_script = render(self._connection_info.redacted)
+        script = template.render(code=code, error_filepath=error_file.relative_to_container)
 
-        script = render(self._connection_info)
         temporary_script_file = self._container.path("/tmp/mysqlsh_script.py")
-        error_file = self._container.path("/tmp/mysqlsh_error.json")
         temporary_script_file.write_text(script)
+
         try:
-            self._container.run_mysql_shell([
-                "--no-wizard",
-                "--python",
-                "--file",
-                str(temporary_script_file.relative_to_container),
-            ])
+            # https://bugs.mysql.com/bug.php?id=117429 details on why --no-wizard is omitted
```

Comment:
should we remove the bug link, since Oracle state it as ["not a bug"](https://bugs.mysql.com/bug.php?id=117429) 
_Though still an undocumented behavior_

Reply:
i kept it (on carl's recommendation) as to why `--no-wizard` is not used. happy to remove it, but at this point, it seems like personal preference

Reply:
User input, as we are doing it (stdin) is more secure (more complex to tamper) and ephemeral.